src/webots/maths/WbMathsUtilities.hpp

```diff
@@ -74,20 +74,20 @@ inline double WbMathsUtilities::normalizeAngle(double angle, double lastSpot = 0
 // If not then the behavior is similar to a clamped value.
 inline double WbMathsUtilities::clampedAcos(double value) {
   assert((fabs(value) < 1.0 + EPSILON) && "Value passed to clampedAcos out of range.");
-  if (value >= 1.0)
+  if (value > 1.0)
```

Comment:
I would keep the `=` in the comparisons because it slightly speeds up the function by saving a call to the acos function.